kml/serdes_gpx.cpp

```diff
@@ -349,6 +349,8 @@ std::string GpxParser::BuildDescription()
     return m_comment;
   else if (m_comment.empty())
     return m_description;
+  else if (m_description == m_comment)
```

Comment:
Great! Let's also just in case do `strings::Trim` on m_description and m_comment when they are initialized from the gpx.

Reply:
> Great! Let's also just in case do `strings::Trim` on m_description and m_comment when they are initialized from the gpx.

I had an idea to do it when I write a code to add them with '\n\n' but looks like xml parser trims elements and no additional work is required (see test kml/kml_tests/gpx_tests.cpp:236)

Reply:
Interesting, I don't see any trim code in expat. Can you please double-check it? Also, our trim removes other characters listed here: https://en.cppreference.com/w/cpp/string/byte/isspace

Reply:
You are right, looks like we already trim all CharData here: kml/serdes_gpx.cpp:276, so no additional trim for desc/cmt is required.

Reply:
> Interesting, I don't see any trim code in expat. Can you please double-check it? Also, our trim removes other characters listed here: https://en.cppreference.com/w/cpp/string/byte/isspace

You are right, looks like we already trim all CharData here: kml/serdes_gpx.cpp:276, so no additional trim for desc/cmt is required.